index.js

```diff
@@ -150,7 +154,7 @@ declare class Cjdnsniff_CtrlMsg extends Cjdnsniff_GenericMsg {
 export type Cjdnsniff_CtrlMsg_t = Cjdnsniff_CtrlMsg;
 export type Cjdnsniff_t = {
     on: (event: string, listener: (any)=>void) => Cjdnsniff_t,
-    send: (Object, ()=>void) => void
+    send: (Object, string, ()=>void) => void
```

Comment:
ahh ok, shouldn't matter much since we'll just use the new API in future code but flow will enforce not using it the old way. Fine for me.